src/index.js

```diff
@@ -8,6 +8,7 @@ import RedBox from './components/RedBox';
 import View from './components/View';
 import Text from './components/Text';
 import TextStyles from './sharedStyles/TextStyles';
+import Symbol from './symbol';
```

Comment:
oh shoot! i forgot to reference the original issue that this proposes to resolve! 0 for 2 on clarity so far 😆  anyway i'll change it to `Symbols`. thanks!